### ISSUE TITLE
how to stop Early stop function 

### ISSUE THREAD

**Issue by JustinNober** [ultralytics/yolov5#5288]

**Comment by glenn-jocher** [comment]
@JustinNober you can disable early stopping by passing 0, or the same value as `--epochs` for `--patience`:
```bash
python train.py --patience 0  # EarlyStopping disabled
python train.py --patience 300 --epochs 300  # EarlyStopping will not happen
```

**Comment by glenn-jocher** [comment]
@JustinNober good news 😃! Your original issue may now be fixed ✅ in PR #5303. This PR implements a more informative EarlyStopping message explaining the reason for stopping and steps you can take to update or disable EarlyStopping behavior.

To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload with `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!

**Comment by chiragpandav** [comment]
usage: train.py [-h] [--model MODEL] [--data DATA] [--epochs EPOCHS]
                [--batch-size BATCH_SIZE] [--imgsz IMGSZ] [--nosave]
                [--cache [CACHE]] [--device DEVICE] [--workers WORKERS]
                [--project PROJECT] [--name NAME] [--exist-ok]
                [--pretrained [PRETRAINED]]
                [--optimizer {SGD,Adam,AdamW,RMSProp}] [--lr0 LR0]
                [--decay DECAY] [--label-smoothing LABEL_SMOOTHING]
                [--cutoff CUTOFF] [--dropout DROPOUT] [--verbose]
                [--seed SEED] [--local_rank LOCAL_RANK]


train.py: error: unrecognized arguments: --patience 2

**Comment by glenn-jocher** [comment]
@chiragp007 hi there! It seems that the `train.py` script does not have a `--patience` argument. The available arguments for `train.py` are listed in the usage section below:

```bash
usage: train.py [-h] [--model MODEL] [--data DATA] [--epochs EPOCHS]
                [--batch-size BATCH_SIZE] [--imgsz IMGSZ] [--nosave]
                [--cache [CACHE]] [--device DEVICE] [--workers WORKERS]
                [--project PROJECT] [--name NAME] [--exist-ok]
                [--pretrained [PRETRAINED]]
                [--optimizer {SGD,Adam,AdamW,RMSProp}] [--lr0 LR0]
                [--decay DECAY] [--label-smoothing LABEL_SMOOTHING]
                [--cutoff CUTOFF] [--dropout DROPOUT] [--verbose]
                [--seed SEED] [--local_rank LOCAL_RANK]
```

If you have any more questions or need further assistance, please feel free to ask.
